backend/dataSeeder.js

```diff
@@ -0,0 +1,44 @@
+const dotenv = require("dotenv");
```

Comment:
Minor comment: I would place this file into a `/scripts` folder

Reply:
Fixed: Changes are reflected in recent commits